.gitignore

```diff
@@ -52,6 +52,15 @@ ENV/
 # JetBrains
 .idea/
 
+# VS Code
+.vscode
+__pycache__
```

Comment:
`__pycache__` is already covered by line 3 in this file. This line is unnecessary.